airbyte-cdk/python/unit_tests/sources/file_based/scenarios/scenario_builder.py

```diff
@@ -46,7 +45,10 @@ def __init__(
         expected_read_error: Tuple[Optional[Type[Exception]], Optional[str]],
         incremental_scenario_config: Optional[IncrementalScenarioConfig],
         expected_analytics: Optional[List[AirbyteAnalyticsTraceMessage]] = None,
+        log_levels: Optional[Set[str]] = None,
```

Comment:
allow test developers to configure which logs are captured so we can verify the output of a read